### PR TITLE
Feat: add documentation for What is Tequila API?

### PR DESCRIPTION
**Pull request by FilipMessa**

### CODE REVIEW

File: docs/tequila-api.md [hunk 1]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
 sidebar_label: What is Tequila API?
 ---
 
-_This page is a stub._
+With the [Tequila](https://tequila.kiwi.com) platform, you can build your o travel business application for free, with the [kiwi.com](https://www.kiwi.com)'s content, technology, and services.
```
Comment (aalises) [inline docs/tequila-api.md]: `You can build your travel business application` without the o

File: docs/tequila-api.md [hunk 2]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
 
-Tequila API is an API.
+All you need to start working with the **Tequila** API is you need to be registered and have your API key.
```
Comment (aalises) [inline docs/tequila-api.md]: `with the Tequila API is to be registered`

File: docs/tequila-api.md [hunk 3]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
+Check the Guide: [Get your Tequila API key](guide-tequila-api-key.md) for information on how to grab your API Key.
+
+To implement the basic search, you need the `/locations` and `/search` API calls.
+
+- **Locations API**: search for the cities, airports, stops, and other locations.([docs](https://tequila.kiwi.com/portal/resources/locations_api))
```
Comment (aalises) [inline docs/tequila-api.md]: `search for cities, airports...`

File: docs/tequila-api.md [hunk 4]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
+
+- **Search API**: this API allows you search for one-way or return type of itineraries.([docs](https://tequila.kiwi.com/portal/resources/search_api))
```
Comment (aalises) [inline docs/tequila-api.md]: `allows you to search`

File: docs/tequila-api.md [hunk 5]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
+
+- **Mutlicity API**: this API provides the multicity search. The multicity trip is a journey when the passenger travels from one city to another, stays there for a couple of days and then continues the journey to another city. ([docs](https://tequila.kiwi.com/portal/resources/multicity_api))
```
Comment (aalises) [inline docs/tequila-api.md]: `a journey where the passenger...`

File: docs/tequila-api.md [hunk 6]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
+
+- **NOMAD API**: this is search API is for those travelers, who would like to visit n-number of cities in any order. ([docs](https://tequila.kiwi.com/portal/resources/nomad_api))
```
Comment (aalises) [inline docs/tequila-api.md]: `This search API is for those..`, `like to visit a n-number of cities`

File: docs/tequila-api.md [hunk 7]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
+
+For finish to booking itinerary, you will need to call the `/booking` API.
+
+- **Booking API**: methods that allow a user to book the desired itinerary. With this API you will check if the price of is still valid and if the itinerary is still available for the booking. Save the booking which initiates the booking on our backend and confirms that the user payment was successfully. ([docs](https://tequila.kiwi.com/portal/resources/booking_api))
```
Comment (aalises) [inline docs/tequila-api.md]: `This API allows you to check if the price of the itinerary is still valid and if the itinerary is still available`. `It also saves the booking which initiates the booking process...`, `the user payment was successful`

File: docs/tequila-api.md [hunk 8]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
+
+- **Locations API**: search for the cities, airports, stops, and other locations.([docs](https://tequila.kiwi.com/portal/resources/locations_api))
+
+- **Search API**: this API allows you search for one-way or return type of itineraries.([docs](https://tequila.kiwi.com/portal/resources/search_api))
+
+- **Mutlicity API**: this API provides the multicity search. The multicity trip is a journey when the passenger travels from one city to another, stays there for a couple of days and then continues the journey to another city. ([docs](https://tequila.kiwi.com/portal/resources/multicity_api))
+
+- **NOMAD API**: this is search API is for those travelers, who would like to visit n-number of cities in any order. ([docs](https://tequila.kiwi.com/portal/resources/nomad_api))
+
+For finish to booking itinerary, you will need to call the `/booking` API.
```
Comment (aalises) [inline docs/tequila-api.md]: `In order to finish booking the itinerary, you need to...`

File: docs/tequila-api.md [hunk 9]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
+
+- **Booking API**: methods that allow a user to book the desired itinerary. With this API you will check if the price of is still valid and if the itinerary is still available for the booking. Save the booking which initiates the booking on our backend and confirms that the user payment was successfully. ([docs](https://tequila.kiwi.com/portal/resources/booking_api))
+
+- **Webhooks API**: is a way of how we provide other applications with real-time information. ([docs](https://tequila.kiwi.com/portal/resources/webhooks))
+
+- **VISA API**: with this API you can check if a passenger will need a VISA for a specific country.
+
+More information and documentation about **Tequila API** you can find on the Tequila [resources](https://tequila.kiwi.com/portal/resources) page.
```
Comment (aalises) [inline docs/tequila-api.md]: `You can find more information and documentation about the Tequila API on the Tequila...`

File: docs/tequila-api.md (hunk 5 above)
Comment (FilipMessa) [reply docs/tequila-api.md]: I am not sure if this is better, so must ask for another opinion.
@RobinCsl what do you think?  `a journey where the passenger...` or `a journey when the passenger...`

File: docs/tequila-api.md (hunk 1 above)
Comment (FilipMessa) [reply docs/tequila-api.md]: Thanks, will change

File: docs/tequila-api.md (hunk 2 above)
Comment (FilipMessa) [reply docs/tequila-api.md]: Thanks, will change

File: docs/tequila-api.md (hunk 3 above)
Comment (FilipMessa) [reply docs/tequila-api.md]: Thanks, will change

File: docs/tequila-api.md (hunk 4 above)
Comment (FilipMessa) [reply docs/tequila-api.md]: Thanks, will change

File: docs/tequila-api.md (hunk 9 above)
Comment (FilipMessa) [reply docs/tequila-api.md]: thanks

File: docs/tequila-api.md (hunk 8 above)
Comment (FilipMessa) [reply docs/tequila-api.md]: thanks

File: docs/tequila-api.md (hunk 7 above)
Comment (FilipMessa) [reply docs/tequila-api.md]: thanks

File: docs/tequila-api.md (hunk 6 above)
Comment (FilipMessa) [reply docs/tequila-api.md]: thanks

File: docs/tequila-api.md (hunk 5 above)
Comment (aalises) [reply docs/tequila-api.md]: @FilipMessa I think the correct way is `A journey where + something`

Comment (FilipMessa) [reply docs/tequila-api.md]: OK, thanks I will change as you suggest

Comment (aalises) [reply docs/tequila-api.md]: Perfect! I think it's all good, only one small thing, there's a typo in `Mutlicity API`, with that I think it's good to go 💃

File: docs/tequila-api.md [hunk 10]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
 sidebar_label: What is Tequila API?
 ---
 
-_This page is a stub._
+With the [Tequila](https://tequila.kiwi.com) platform, you can build your travel business application for free, with the [kiwi.com](https://www.kiwi.com)'s content, technology, and services.
```
Comment (RobinCsl) [inline docs/tequila-api.md]:
```suggestion
With the [Tequila](https://tequila.kiwi.com) platform, you can build your travel business application for free, with [kiwi.com](https://www.kiwi.com)'s content, technology, and services. 
```

File: docs/tequila-api.md [hunk 11]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
 
-Tequila API is an API.
+All you need to start working with the **Tequila** API is to be registered and have your API key.
+Check the Guide: [Get your Tequila API key](guide-tequila-api-key.md) for information on how to grab your API Key.
```
Comment (RobinCsl) [inline docs/tequila-api.md]:
```suggestion
Check the guide [Get your Tequila API key](guide-tequila-api-key.md) for information on how to grab your API Key.
```

File: docs/tequila-api.md [hunk 12]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
+
+To implement the basic search, you need the `/locations` and `/search` API calls.
+
+- **Locations API**: Search for cities, airports, stops, and other locations.([docs](https://tequila.kiwi.com/portal/resources/locations_api))
+
+- **Search API**: This API allows you to search for one-way or return type of itineraries.([docs](https://tequila.kiwi.com/portal/resources/search_api))
```
Comment (RobinCsl) [inline docs/tequila-api.md]:
```suggestion
- **Search API**: Search for one-way or return itineraries. ([docs](https://tequila.kiwi.com/portal/resources/search_api))
```

File: docs/tequila-api.md [hunk 13]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
-_This page is a stub._
+With the [Tequila](https://tequila.kiwi.com) platform, you can build your travel business application for free, with the [kiwi.com](https://www.kiwi.com)'s content, technology, and services.
 
-Tequila API is an API.
+All you need to start working with the **Tequila** API is to be registered and have your API key.
+Check the Guide: [Get your Tequila API key](guide-tequila-api-key.md) for information on how to grab your API Key.
+
+To implement the basic search, you need the `/locations` and `/search` API calls.
+
+- **Locations API**: Search for cities, airports, stops, and other locations.([docs](https://tequila.kiwi.com/portal/resources/locations_api))
```
Comment (RobinCsl) [inline docs/tequila-api.md]:
```suggestion
- **Locations API**: Search for cities, airports, stops, and other locations. ([docs](https://tequila.kiwi.com/portal/resources/locations_api))
```

File: docs/tequila-api.md [hunk 14]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
+
+- **Search API**: This API allows you to search for one-way or return type of itineraries.([docs](https://tequila.kiwi.com/portal/resources/search_api))
+
+- **Multicity API**: This API provides the multicity search. The multicity trip is a journey where the passenger travels from one city to another, stays there for a couple of days and then continues the journey to another city. ([docs](https://tequila.kiwi.com/portal/resources/multicity_api))
```
Comment (RobinCsl) [inline docs/tequila-api.md]:
```suggestion
- **Multicity API**: This API allows you to search multi-city itineraries. A _multi-city trip_ is a journey where the passenger travels from one city to another, stays there for a couple of days and then continues the journey to another city. ([docs](https://tequila.kiwi.com/portal/resources/multicity_api))
```

File: docs/tequila-api.md [hunk 15]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
+
+- **NOMAD API**: This search API is for those travelers, who would like to visit n-number of cities in any order. ([docs](https://tequila.kiwi.com/portal/resources/nomad_api))
```
Comment (RobinCsl) [inline docs/tequila-api.md]:
```suggestion
- **NOMAD API**: This search API is for those travellers who would like to visit n-number of cities in any order. ([docs](https://tequila.kiwi.com/portal/resources/nomad_api))
```

File: docs/tequila-api.md [hunk 16]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
+
+In order to finish booking the itinerary, you need to call the `/booking` API.
+
+- **Booking API**: This API allows you to check if the price of the itinerary is still valid and if the itinerary is still available. It also saves the booking which initiates the booking process and confirms that the user payment was successfully. ([docs](https://tequila.kiwi.com/portal/resources/booking_api))
```
Comment (RobinCsl) [inline docs/tequila-api.md]:
```suggestion
- **Booking API**: This API allows you to check if the price of the itinerary is still valid and if the itinerary is still available. It also saves the booking which initiates the booking process and confirms that the user payment was successful. ([docs](https://tequila.kiwi.com/portal/resources/booking_api))
```

File: docs/tequila-api.md [hunk 17]
```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
...
+
+- **Webhooks API**: This API is a way of how we provide other applications with real-time information. ([docs](https://tequila.kiwi.com/portal/resources/webhooks))
```
Comment (RobinCsl) [inline docs/tequila-api.md]:
```suggestion
- **Webhooks API**: This API is a way for <span>Kiwi</span>.com to provide other applications with real-time information. ([docs](https://tequila.kiwi.com/portal/resources/webhooks))
```
I find the documentation on Tequila very un-informative and it's difficult to understand what it's about.

**Pull request merged**